src/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(imCommon_SOURCES
     imLog.cpp
     imString.cpp
     Main.cpp
-)
+        graphics/imPipeline.cpp graphics/imPipeline.h)
```

Comment:
These are indented incorrectly, and the header should be in the imCommon_HEADERS source group.